website/docs/tooling/linter.md

```diff
@@ -352,6 +352,14 @@ const messages = defineMessages({
 })
 ```
 
+### `no-explicit-icu-plural`
+
+Messages that look like `{thing, plural, one {1 thing} other {# things}}` will need to be changed to `{thing, plural, one {# thing} other {# things}}`
+
+#### Why
+
+- one is a category for any number that behaves like 1. So in some languages, for example, one → numbers that end in 1 (like 1, 21, 151) but that don’t end in 11 (like 11, 111, 10311). [More info](https://formatjs.io/docs/core-concepts/icu-syntax/#plural-format)
```

Comment:
why not give some example languages? To my knowledge this is mostly Slavic languages, which includes Russian, Polish, and Ukranian, and probably a bunch of others I don't know off the top of my head

Reply:
totally agree. thanks for the feedback